File: 1048.py
salario = float(input())
if salario < 400.01:
    percent = salario * (15/100)
    reajuste = salario + percent
    print(f'''Novo salario: {reajuste:.2f}
Reajuste ganho: {percent:.2f}
Em percentual: 15 %''')

elif salario < 800.01:
    percent = salario * (12/100)
    reajuste = salario + percent
    print(f'''Novo salario: {reajuste:.2f}
Reajuste ganho: {percent:.2f}
Em percentual: 12 %''')

elif salario < 1200.01:
    percent = salario * (10/100)
    reajuste = salario + percent
    print(f'''Novo salario: {reajuste:.2f}
Reajuste ganho: {percent:.2f}
Em percentual: 10 %''')

elif salario <= 2000:
    percent = salario * (7/100)
    reajuste = salario + percent
    print(f'''Novo salario: {reajuste:.2f}
Reajuste ganho: {percent:.2f}
Em percentual: 7 %''')

elif salario > 2000:
    percent = salario * (4/100)
    reajuste = salario + percent
    print(f'''Novo salario: {reajuste:.2f}
Reajuste ganho: {percent:.2f}
Em percentual: 4 %''')

File: 1048.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var salario = Number(lines.shift());

if (salario < 400.01){
    var percent = salario * (15.00/100.00)
    var reajuste = salario + percent
    console.log(`Novo salario: ${reajuste.toFixed(2)}
Reajuste ganho: ${percent.toFixed(2)}
Em percentual: 15 %`)}

else if (salario < 800.01){
    var percent = salario * (12.00/100.00)
    var reajuste = salario + percent
    console.log(`Novo salario: ${reajuste.toFixed(2)}
Reajuste ganho: ${percent.toFixed(2)}
Em percentual: 12 %`)}

else if (salario < 1200.01){
    var percent = salario * (10.00/100.00)
    var reajuste = salario + percent
    console.log(`Novo salario: ${reajuste.toFixed(2)}
Reajuste ganho: ${percent.toFixed(2)}
Em percentual: 10 %`)}

else if (salario <= 2000){
    var percent = salario * (7.00/100.00)
    var reajuste = salario + percent
    console.log(`Novo salario: ${reajuste.toFixed(2)}
Reajuste ganho: ${percent.toFixed(2)}
Em percentual: 7 %`)}

else if (salario > 2000){
    percent = salario * (4/100)
    reajuste = salario + percent
    console.log(`Novo salario: ${reajuste.toFixed(2)}
Reajuste ganho: ${percent.toFixed(2)}
Em percentual: 4 %`)}
